src/analysis/calculus/fderiv.lean

```diff
@@ -580,8 +581,10 @@ lemma differentiable_on_const (c : F) : differentiable_on 𝕜 (λx, c) s :=
 
 end const
 
-/- Bounded linear maps -/
-section is_bounded_linear_map
+/- Continuous linear maps. There are currently two variants of these in mathlib, the bundled version
```

Comment:
We haven't updated the doc guidelines yet, but the doc generator can now read module docs in the middle of the file.
```suggestion
/- ### Continuous linear maps

There are currently two variants of these in mathlib, the bundled version
```

Reply:
Does that mean you also need `/-!` with the `!` bang?

Reply:
I have commited (without the `!`). @robertylewis , if it is in fact needed tell me! Anyway, thanks for the news, improving the docs is in my opinion very important and useful!

Reply:
Oops, yeah, it needs to be `/-!`. My mistake. https://robertylewis.com/mathlib_docs/topology/whoa.html is an example of what it looks like right now. (Once this and #1715 land, we'll have a few real examples.)

Reply:
I added it.